src/main/features/testing-support/routes/create-claim-draft.ts

```diff
@@ -29,6 +32,8 @@ export default express.Router()
       draft.document = new DraftClaim().deserialize(prepareClaimDraft(user.email))
       await new DraftService().save(draft, user.bearerToken)
 
+      await claimStoreClient.addRoleToUser(user, 'cmc-new-features-consent-given')
```

Comment:
This doesn't check if the user has already given consent or withheld their consent?
I.e. a user who says no to consent should still be able to use this endpoint and not be opted in